projects/client/RabbitMQ.Client/RabbitMQ.Client.csproj

```diff
@@ -1,42 +1,47 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <VersionPrefix>6.0.0</VersionPrefix>
     <TargetFrameworks>net461;netstandard2.0</TargetFrameworks>
     <NoWarn>$(NoWarn);CS1591</NoWarn>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <AssemblyTitle>RabbitMQ Client Library for .NET</AssemblyTitle>
+    <Authors>VMware</Authors>
+    <Company>VMware, Inc. or its affiliates.</Company>
+    <Copyright>Copyright © 2007-2020 VMware, Inc. or its affiliates.</Copyright>
+    <Description>The RabbitMQ .NET client is the official client library for C# (and, implicitly, other .NET languages)</Description>
+    <EmbedUntrackedSources>true</EmbedUntrackedSources>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <PackageTags>RabbitMQ</PackageTags>
     <PackageIcon>icon.png</PackageIcon>
+    <PackageLicenseFile>LICENSE</PackageLicenseFile>
     <PackageProjectUrl>https://www.rabbitmq.com/dotnet.html</PackageProjectUrl>
-    <PackageLicense>https://www.rabbitmq.com/dotnet.html</PackageLicense>
-    <AssemblyTitle>RabbitMQ Client Library for .NET</AssemblyTitle>
-    <Description>The RabbitMQ .NET client is the official client library for C# (and, implicitly, other .NET languages)</Description>
-    <Company>VMware, Inc. or its affiliates.</Company>
+    <PackageTags>rabbitmq, amqp</PackageTags>
     <Product>RabbitMQ</Product>
-    <Copyright>Copyright © 2007-2020 VMware, Inc. or its affiliates.</Copyright>
-    <GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
     <PublishRepositoryUrl>true</PublishRepositoryUrl>
-    <EmbedUntrackedSources>true</EmbedUntrackedSources>
     <IncludeSymbols>true</IncludeSymbols>
     <SymbolPackageFormat>snupkg</SymbolPackageFormat>
     <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
     <AssemblyOriginatorKeyFile>../rabbit.snk</AssemblyOriginatorKeyFile>
     <SignAssembly>true</SignAssembly>
+    <MinVerTagPrefix>v</MinVerTagPrefix>
+    <MinVerVerbosity>detailed</MinVerVerbosity>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <PackageOutputPath>..\..\..\packages</PackageOutputPath>
   </PropertyGroup>
 
   <ItemGroup>
     <Compile Include="..\..\..\gensrc\autogenerated-api-0-9-1.cs" />
-  </ItemGroup>
-
-  <ItemGroup>
     <None Remove="icon.png" />
     <Content Include="icon.png" PackagePath="" />
-  </ItemGroup>
-
-  <ItemGroup>
```

Comment:
I personally like seeing the package references group separately into their own ItemGroup instead of all being in one, but that's more style / readability than a technical reason.

Reply:
Feel free to re-org however you'd like! I'll refrain from smashing things all together going forward 😄

Reply:
> Feel free to re-org however you'd like! I'll refrain from smashing things all together going forward 😄

Just wanted to make sure I was pointing out style / preference things as such, and not trying to dictate how to do things!